app/src/main/java/org/oppia/app/player/audio/LanguageInterface.kt

```diff
@@ -7,4 +7,10 @@ interface LanguageInterface {
 
   /** Open the language selection dialog. */
   fun languageSelectionClicked()
+
+  /** Getter for userIsSeeking in AudioFragment */
```

Comment:
Unlike the shim interfaces, this one is longer-lived. Let's describe what implementations should actually do for this method, e.g.: ``Returns whether the user is actively seeking to a new audio position, that is, dragging the knob to a new position in the audio track.``

Ditto for the method below.

Reply:
Ah - will do. Thank you!

Reply:
Okay!

Reply:
Much clearer--thanks!